packages/xsnap/package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "repl": "node src/xsrepl.js",
     "build:bin": "if test -d ./test; then node src/build.js; else yarn build:from-env; fi",
-    "build:env": "test -d ./test && node src/build.js --show-env > build.env",
+    "build:env": "node src/build.js --show-env > build.env",
```

Comment:
In both the old and new `build:env` script, we fail if `git` is unavailable, since that is a requirement for `--show-env`, but now `src/build.js` says why. In the old code, we would short circuit if `test` didn’t exist, which is the case for an unpacked npm tarball, an environment where `git` _might_ not exist and in which `git` should not be used. In the new code, we use the existence of a specific source file and the platform-specific binary for a more direct signal.